### ISSUE TITLE
kubectl-version input arg is not documented

### ISSUE THREAD

**Issue by stigok** [Azure/k8s-deploy#187]
I saw the `kubectl-version` argument being used in [one of the examples](https://github.com/Azure/k8s-deploy#basic-deployment-without-any-deployment-strategy) here in the readme, however, it's not documented in the list of input arguments.

**Comment by OliverMKing** [comment]
Good catch! This isn't a feature of this action anymore. That should be deleted.
